athf/core/attack_matrix.py

```
@@ -90,9 +123,336 @@ class TacticInfo(TypedDict):
+        self._ensure_loaded()
+        # Try to extract version from STIX bundle
+        try:
+            with open(str(self._stix_path), "r") as f:
+                bundle = json.load(f)
+            for obj in bundle.get("objects", []):
+                if obj.get("type") == "x-mitre-collection":
+                    version = obj.get("x_mitre_version", "")
+                    if version:
+                        return f"v{version} (STIX)"
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)